validate/validate.go

```diff
@@ -109,9 +109,20 @@ func (v *Validator) CheckAll() (msgs []string) {
 	return
 }
 
-// CheckRootfsPath checks status of v.spec.Root.Path
-func (v *Validator) CheckRootfsPath() (msgs []string) {
-	logrus.Debugf("check rootfs path")
+// CheckRootfs checks status of v.spec.Root
+func (v *Validator) CheckRootfs() (msgs []string) {
+	logrus.Debugf("check rootfs")
```

Comment:
`rootfs` is the value that the spec recommends for POSIX `root.path`.  But since this method is also checking `root.readonly`, I'd recommend using `CheckRoot` instead of `CheckRootfs`, `"check root"` instead of `"check rootfs"`, etc.